docs/governance-and-management/governance/governance-processes.md

```diff
@@ -66,3 +66,30 @@ Types of technical design decisions that require approval:
     |**Product Design**|Consulted|Consulted|Consulted|Consulted|
     |**Technical Design**|Responsible|Responsible|Responsible|Accountable|
 
+### Technical Design Process
+
+This section outlines the process for features that require a technical design document. The purpose of a design document is to draft and approve major technical design decisions to ensure the quality, reliability, and maintainability of CiviForm for all its users.
+
+#### Before You Start
+
+The following should be true:
+1. There is an open issue in [GitHub](https://github.com/civiform/civiform/issues)
+2. There is an approved PRD in the [public folder)(https://drive.google.com/drive/u/0/folders/1gqw3VSK-lMPgRfcS_ZEhF3C_51We0eGz)
+3. If the issue has a `UX` label, approved UX Mocks are added to the [public folder](https://drive.google.com/drive/u/0/folders/1zY6OnQYH7Xo5ebPctTJtd5QRsHBAI5vF)
+4. A `Ready for TDD` label is added to the issue and visible in [this list](https://github.com/civiform/civiform/issues?q=is%3Aissue+is%3Aopen+label%3AReady+for+TDD)
```

Comment:
question (blocking): it looks like the label is unquoted in the search query which may not work as expected: 
![image](https://user-images.githubusercontent.com/473671/205188404-acaf2786-2c06-4a37-958f-3b5ce3afa68d.png)

Reply:
Updated link